### PR TITLE
Add Relative Volume (RVOL) Indicator

### LINKED ISSUE #4165: Relative Volume (RVOL) Indicator
#### Expected Behavior
Support "Relative Volume (RVOL)" Indicator
https://www.smbtraining.com/blog/relative-volume-rvol-defined-and-how-we-use-it

#### Actual Behavior
No implementation of "Relative Volume (RVOL)" indicator

#### Potential Solution
Implement "Relative Volume (RVOL)" Indicator

#### Checklist
- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)

### PR DESCRIPTION
**Pull request by abolfazlghalandary**
#### Description
Since I'm new to contributing, I will appreciate any step to step help to make this PR applicable for merging.

#### Related Issue
Closes #4165

#### Types of changes

- [x] New feature (non-breaking change which adds functionality)

#### Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [x] I have added tests to cover my changes. 
- [ ] All new and existing tests passed.
- [x] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

**Comment by abolfazlghalandary** [comment]
I used RelativeDailyVolume tests and the Data Set attached to the issue. Are the tests OK?
Could you please guide on how to pass the tests? I can't find out how the data set calculated the RVOL value.

File: Indicators/RelativeVolume.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        public override bool IsReady => RollingSum.IsReady;
+
+        /// <summary>
+        /// Resets this indicator to its initial state
+        /// </summary>
+        public override void Reset()
+        {
+            RollingSum.Reset();
+            base.Reset();
+        }
```
Comment (jhonabreul) [inline Indicators/RelativeVolume.cs:42]: Usually in Lean constructors are placed at the top, after attributes and properties, and before any other methods.

**Review by jhonabreul — CHANGES_REQUESTED** [review]
Thank you for your contribution! 😁 

Just leaving a few comments for you to get this merge ready and help you with the indicator testing. I'll be here if you have any more questions.

**Comment by Martin-Molinero** [comment]
Hello @abolfazlghalandary! It seems we've missed detecting that this indicator was already implemented, see comment https://github.com/QuantConnect/Lean/issues/4165#issuecomment-1442172231, sorry for the trouble.
There are other good first issues we've tagged if your interested https://github.com/QuantConnect/Lean/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Pull request closed without merging**